test/unittest/blas3/blas3_trsm_test.cpp

```diff
@@ -84,7 +92,7 @@ void run_test(const combination_t<scalar_t> combi) {
       blas::helper::copy_to_host<scalar_t>(q, b_gpu, B.data(), B.size());
   sb_handle.wait(event);
 
-  bool isAlmostEqual = utils::compare_vectors(cpu_B, B);
+  bool isAlmostEqual = utils::compare_vectors(cpu_B, B, std::cerr, "", true);
```

Comment:
Thanks for your PR, @muhammad-tanvir-1211.

```suggestion
  bool isAlmostEqual = utils::compare_vectors(cpu_B, B, std::cerr, "", en_joint_matrix != NULL && en_joint_matrix == '1');
```
I think it is better to increase the tolerance just in case of the joint matrix is used.